doc/sphinx/source/guides/getting-started.rst

```diff
@@ -67,7 +67,7 @@ Using Nix
 #. Run Nix shell to get all environment required::
 
      # Run from theCore directory
-     nix-shell --pure
+     nix-shell .
```

Comment:
`.` is not required

Reply:
Let's leave it there to be more explicit about importance of current working dir.